frappe/email/smtp.py

```diff
@@ -59,7 +59,7 @@ def get_outgoing_email_account(raise_exception_not_set=True, append_to=None):
 
 		if email_account:
 			if email_account.enable_outgoing and not getattr(email_account, 'from_site_config', False):
-				email_account.password = email_account.get_password()
+				email_account.password = email_account.get_password(raise_exception=False)
```

Comment:
`raise_exception = False if email_account.smtp_server in ('localhost', '127.0.0.1') or True`